google/cloud/security/inventory/inventory_loader.py

```diff
@@ -47,7 +47,7 @@
 from google.cloud.security.common.data_access import errors as data_access_errors
 from google.cloud.security.common.data_access.dao import Dao
 from google.cloud.security.common.data_access.sql_queries import snapshot_cycles_sql
-from google.cloud.security.common.gcp_api import admin_directory
+from google.cloud.security.common.gcp_api import admin_directory as ad
```

Comment:
@ahoying has requested that we make the import statements consistent and import the module instead of the class, so that's the approach that @blueandgold and I have been taking -- I can't seem to find his comment about why though